.gitignore

```diff
@@ -17,3 +17,6 @@
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore vim swaps
+*.swp
```

Comment:
Please put these kind of ignores in your global .gitignore. See https://help.github.com/articles/ignoring-files/ for more info.